src/axom/core/NumericArray.hpp

```diff
@@ -349,27 +349,32 @@ class NumericArray  // NOLINT
    * \brief Find the max component.
    * \return The value of the largest component.
    */
+  AXOM_HOST_DEVICE
```

Comment:
Could this be on the same line to be consistent with the inline definition below?

Reply:
I think the line breaks are entirely determined by the `clang-format` rules.
I'll manually move it and see if it reverts.

Reply:
clang-format was agnostic about this change (`AXOM_HOST_DEVICE` on the same line or split across two lines).
I converted everything to be on the same line.